src/encoded/tests/test_schema_target.py

```diff
@@ -187,3 +195,26 @@ def test_target_organism_uuid(testapp, human):
     }
     res = testapp.post_json('/target', item).json['@graph'][0]
     assert res['organism'] == human['@id']
+
+
+def test_synthetic_tag_target(testapp, target_synthetic_tag, human, ctcf):
+    flag_target = testapp.post_json(
+        '/target', target_synthetic_tag
+    ).json['@graph'][0]
+    assert flag_target['title'] == 'FLAG (Synthetic tag)'
+
+    syn_and_org = {'target_organism': human['uuid']}
+    syn_and_org.update(target_synthetic_tag)
+    res = testapp.post_json('/target', syn_and_org, expect_errors=True)
+    assert res.json['status'] == 'error'
```

Comment:
Use more specific error if possible. If just validation error you can say something like `testapp.post_json('/target', syn_and_org, status=422)` instead and get rid of assert statement.

Reply:
Great suggestion! If just for successful posting, I can also get rid of `assert res.json['status'] == 'success'`, right?